strax/storage/mongo.py

```diff
@@ -0,0 +1,137 @@
+"""I/O format for MongoDB
+
+This plugin is designed with data monitoring in mind, to put smaller
+amounds of extracted data into a database for quick access. However
+it should work with any plugin.
+
+Note that there is no check to make sure the 16MB document size
+limit is respected!
+"""
+
+try:
```

Comment:
pymongo is now a dependency (due to using json_util also in the files backend), so this check is not needed.